shared/params/config.go

```diff
@@ -246,6 +246,7 @@ func MinimalSpecConfig() *BeaconChainConfig {
 	minimalConfig.ShuffleRoundCount = 10
 	minimalConfig.MinGenesisActiveValidatorCount = 64
 	minimalConfig.MinGenesisTime = 0
+	minimalConfig.TargetAggregatorsPerCommittee = 1
```

Comment:
Why 1? Just wondering

Reply:
For 64 validators, you get 4 validators per committee. 1 is the easiest number to see it work live given the modulus selection approach